srml/support/src/dispatch.rs

```diff
@@ -726,7 +728,7 @@ macro_rules! decl_module {
 							)
 						},
 					)*
-					$call_type::__PhantomItem(_) => { panic!("__PhantomItem should never be used.") },
+					$call_type::__PhantomItem(_, _) => { panic!("__PhantomItem should never be used.") },
```

Comment:
Maybe replace `panic!` with `unreachable!` for consistency here? I suppose the panic error message is useless if `__PhantomItem` cannot be constructed.